### PR TITLE
clusteroperator pkg: switch to the goclient.Client, Get method added + unit test

### PR DESCRIPTION
**Pull request by elenagerman**
switch to the goclient.Client, Get method added + unit test

### CODE REVIEW

**Review by sebrandon1 — APPROVED** [review]

**Comment by elenagerman** [comment]
@kononovn @trewest @achuzhoy PTAL

**Comment by elenagerman** [comment]
@kononovn @trewest @achuzhoy PTAL

File: pkg/clusteroperator/clusteroperator.go [hunk 1]
```diff
@@ -25,11 +28,11 @@ const (
 // Builder provides struct for clusterOperator object.
 type Builder struct {
 	// ClusterOperator definition. Used to create a clusterOperator object.
-	Definition *v1.ClusterOperator
+	Definition *configV1.ClusterOperator
```
Comment (kononovn) [inline pkg/clusteroperator/clusteroperator.go:30]: Please refactor it to *configv1.ClusterOperator

**Review by kononovn — CHANGES_REQUESTED** [review]
Looks good just 2 comments

File: pkg/clusteroperator/clusteroperator.go (hunk 1 above)
Comment (elenagerman) [reply pkg/clusteroperator/clusteroperator.go:30]: but for the clint.go it is configV1, why?

Comment (kononovn) [reply pkg/clusteroperator/clusteroperator.go:30]: We'll refactor clients.go other packages later. Now we are trying to switch to small letters in import

**Review by kononovn — CHANGES_REQUESTED** [review]
1 comment to fix and we can merge it.

**Review by kononovn — APPROVED** [review]
/lgtm

**Pull request merged**